app/templates/service.html

```diff
@@ -93,6 +97,11 @@ <h2 class="govuk-heading-m app-summary-list-heading" id="service-documents">Serv
       </li>
     {% endfor %}
     </ul>
+    {{ govukDetails({
+      "classes": "govuk-!-font-size-16",
+      "summaryText": "Request an accessible format",
+      "html": 'If you use assistive technology (such as a screen reader) and need a version of these documents in a more accessible format, please email <a href="mailto:info@crowncommercial.gov.uk" target="_blank" class="govuk-link">info@crowncommercial.gov.uk</a>. Please tell us what format you need. It will help us if you say what assistive technology you use.'
```

Comment:
This line is missing the `span` from the other details component.

```suggestion
      "html": 'If you use assistive technology (such as a screen reader) and need a version of these documents in a more accessible format, please email <span class="govuk-!-display-block govuk-!-margin-top-1 govuk-!-margin-bottom-2"><a href="mailto:info@crowncommercial.gov.uk" target="_blank" class="govuk-link">info@crowncommercial.gov.uk</a>.</span> Please tell us what format you need. It will help us if you say what assistive technology you use.'
```

Is it possible to put the details component (or even the whole list of documents) in a template partial?

Reply:
This was on purpose. The span'd version is for the small meta column, to make the email stand out without weird line breaks:
![Screenshot 2020-12-08 at 12 28 32](https://user-images.githubusercontent.com/22524634/101484112-08efce80-3951-11eb-8e6b-5002106e6dff.png)

The non-spanned version is for the more generous space at the bottom of the page:
![Screenshot 2020-12-08 at 12 28 25](https://user-images.githubusercontent.com/22524634/101484167-1dcc6200-3951-11eb-9bd2-96689de4af1d.png)

Reply:
I could rejig them to look the same and move this out into a separate file though.

Reply:
Done

Reply:
Ah, sorry, I didn't realise it was on purpose.

Reply:
I think both versions look good when testing them with a mobile form factor or when zoomed in to 400%, so I'd say it's up to you to go with which version you prefer.